crmlogin/Contact.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+Methods used to edit contacts. Used in actualContactEdit.php
+*/
+	class Contact{
+		
+
+		private $userInfo = array('firstname' => " ", 'lastname' => " ", 'email' => " ");
+		private $user; 
+		private $currentEmail;
+
+/**
+Instantiates the object by email
+*/
+		public static function GetByEmail($email){ 
+			include 'connection.php';
+			session_start();
+			global $user;
+			$user = $_SESSION['user'];
+			if(!isset($user)){
+				header("Location: index.html");
+			}
+
+			global $userInfo;
+			global $currentEmail;
+			$sql = "SELECT * FROM `contactlist` WHERE `email`='$email' AND `user`='$user'";
+			$result = $conn->query($sql);
+			$info = $result->fetch_assoc();
+
```

Comment:
Remember we want these static methods to return a Contact object... so it should be something like new Contact in here and the return that contact.